api/src/main/java/org/pmiops/workbench/opsgenie/EgressEventServiceImpl.java

```diff
@@ -84,19 +101,20 @@ private void createEgressEventAlert(EgressEvent egressEvent) {
       logger.info(
           String.format(
               "Successfully created or updated Opsgenie alert for high-egress event on project %s (Opsgenie request ID %s)",
-              egressEvent.getProjectName(), response.getRequestId()));
+              egressEvent.getWorkspaceNamespace(), response.getRequestId()));
     } catch (ApiException e) {
       logger.severe(
           String.format(
               "Error creating Opsgenie alert for egress event on project %s: %s",
-              egressEvent.getProjectName(), e.getMessage()));
+              egressEvent.getWorkspaceNamespace(), e.getMessage()));
       e.printStackTrace();
     }
   }
 
   private CreateAlertRequest egressEventToOpsGenieAlert(EgressEvent egressEvent) {
     final CreateAlertRequest request = new CreateAlertRequest();
-    request.setMessage(String.format("High-egress event (%s)", egressEvent.getProjectName()));
+    request.setMessage(
```

Comment:
may want to add the google project here too ... not sure

Reply:
Feel that is too long in the titile. and namespace is still identical, so I just use namespace here, google project is in description below.

Reply:
Didn't realize this was the msg subject.  Thanks.